samples/sample_decode/src/pipeline_decode.cpp

```diff
@@ -1963,16 +1955,14 @@ mfxStatus CDecodingPipeline::RunDecoding()
             CTimer::ConvertToSeconds(*std::min_element(m_vLatency.begin(), m_vLatency.end()))*1000);
     }
 
-    if (m_eWorkMode == MODE_RENDERING) {
+    if (m_eWorkMode == MODE_RENDERING)
```

Comment:
please, don't change what you actually did not modify.

Reply:
fixed